conf/pipeline/proteinfold/crg.config

```
@@ -16,11 +16,15 @@ profiles {
             withName: 'ARIA2' {
                 time = '12h'
             }
-            withName: 'MMSEQS_COLABFOLDSEARCH' {
-                queue = 'mem_512'
-                memory = "100 GB"
-                cpus = 8
-                time = '12h'
+        }
+        if (params.mode == 'colabfold') {
+            process {
+                withName: 'MMSEQS_COLABFOLDSEARCH' {
+                    queue = 'mem_512'
+                    memory = "100 GB"
+                    cpus = 8
+                    time = '12h'
+                }
```

Maybe add also here the other colabfold process and add a condition for alphafold before line 10?
```suggestion
                }
                withName: 'COLABFOLD_BATCH' {
                    cpus = 1
                    memory = "30 GB"
                    queue = params.use_gpu ? 'gpu' : 'long-sl7'
                    clusterOptions = { ( task.queue == 'gpu' ? '-l gpu=1' : '' ) }
               }

```

---

Yes, sure. I will add the change to my commit